src/cascadia/TerminalSettingsModel/WslDistroGenerator.cpp

```diff
@@ -47,6 +49,16 @@ std::wstring_view WslDistroGenerator::GetNamespace() const noexcept
     return WslGeneratorNamespace;
 }
 
+std::wstring_view WslDistroGenerator::GetDisplayName() const noexcept
+{
+    return RS_(L"WslDistroGeneratorDisplayName");
+}
+
+std::wstring_view WslDistroGenerator::GetIcon() const noexcept
+{
+    return IconPath;

```

Comment:
This icon doesn't look that great since I'm resizing it to make it bigger:
![{15EA8CE4-6E3F-4DFD-8F19-2BE62347326A}](https://github.com/user-attachments/assets/cb9d4468-1563-466e-8c5d-7587d0c3a05b)


Is there a better resource I can use here?